cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -177,6 +177,14 @@
     </description>
   </property>
 
+  <property>
+    <name>namespaces.creation.hook.enabled</name>
```

Comment:
now that this exists, we should remove the logic around `master.environment.k8s.namespace.creation.enabled` in the KubeMasterEnvironment

Reply:
Done